components/Actions/Actions.js

```diff
@@ -1,17 +1,29 @@
-import Action from './Action/Action';
-import styles from './Actions.module.css';
+import Action from "./Action/Action";
+import styles from "./Actions.module.css";
 
-const Actions = ({ actions, hasSubmit, title }) => (
-	<div className={styles.actions}>
-		<h3 className={styles.title}>{title ?? 'Actions'}</h3>
-		<div className={styles.wrapper}>
-			{actions?.map(({ copy }, index) => <Action copy={copy} key={index} />)}
-			{hasSubmit && <Action copy="Propose a new action..." isContact />}
-		</div>
-	</div>
-)
+const Actions = ({ actions, id, markdowns, hasSubmit, title }) => {
+  return (
+    <div className={styles.actions}>
+      <h3 className={styles.title}>{title ?? "Actions"}</h3>
+      <div className={styles.wrapper}>
+        {actions?.map(({ copy }, index) => (
+          <Action
+            copy={copy}
+            key={index}
+            markdown={
+              markdowns.find(
```

Comment:
How large is `markdowns`? Wondering if there is any benefit to memoising this search to avoid running on every render.

Reply:
`markdowns` would contain an array of all copy. It could be a good idea to memoize it.